llama_stack/providers/inline/safety/code_scanner/code_scanner.py

```diff
@@ -5,15 +5,19 @@
 # the root directory of this source tree.
 
 import logging
+import uuid
 from typing import Any
 
+from codeshield.cs import CodeShield, CodeShieldScanResult
```

Comment:
you should add a `if TYPE_CHECKING:` since you need the import only for that. this is a really bad import and pulls in random stuff so that's why we had relegated it to within the method.